README.md

```diff
@@ -1,15 +1,29 @@
 <img src="https://www.ga4gh.org/wp-content/themes/ga4gh/dist/assets/svg/logos/logo-full-color.svg" alt="GA4GH Logo" style="width: 400px;"/>
 
+![release_badge](https://img.shields.io/github/v/tag/ga4gh/task-execution-schemas)
+
+
 Task Execution Service (TES) API
 ======================================
-<sup>`master` branch status: </sup>[![Build Status](https://travis-ci.org/ga4gh/task-execution-schemas.svg?branch=master)](https://travis-ci.org/ga4gh/task-execution-schemas?branch=master)
-<a href="https://raw.githubusercontent.com/ga4gh/task-execution-schemas/master/openapi/task_execution_service.openapi.yaml"><img src="http://online.swagger.io/validator?url=https://raw.githubusercontent.com/ga4gh/task-execution-schemas/master/openapi/task_execution_service.openapi.yaml" alt="Open API Validator" height="20em" width="72em"></A>
+
+This repository is the home for the schema for the GA4GH Task Execution API. The Goal of the API is to provide a standardized way to executing batch execution tasks.
 
 The [Global Alliance for Genomics and Health](http://genomicsandhealth.org/) is an international coalition, formed to enable the sharing of genomic and clinical data.
 
 Cloud Work Stream
 -----------------
 
+The [Cloud Work Stream](https://ga4gh.cloud) helps the genomics and health communities take full advantage of modern
+cloud environments.
+Our initial focus is on “bringing the algorithms to the data”, by creating standards for defining, sharing, and
+executing portable workflows.
+
+We work with platform development partners and industry leaders to develop standards that will facilitate
+interoperability.
+
+Cloud Work Stream
```

Comment:
If I'm not mistaken, it looks like we're gonna have two sections with the same name. If so, please merge into a single section.

Reply:
@uniqueg fixed